opta/commands/apply.py

```diff
@@ -160,7 +160,9 @@ def _apply(
             )
             < 0
         ):
-            raise UserErrors()
+            raise UserErrors(
+                f"Can not run with older version-- last run at {previous_config['date']} with version {old_opta_version}. Please update to that version or later"
```

Comment:
Message could be slightly clarified as `You're trying to run an older version of opta (last run was at <date> with version <version>). Please update to the latest version and try again!`